netbox/utilities/templatetags/helpers.py

```diff
@@ -137,6 +139,39 @@ def tzoffset(value):
     return datetime.datetime.now(value).strftime('%z')
 
 
+@register.filter(expects_localtime=True)
+def date_span(date_value):
```

Comment:
I thought I mentioned this previously, but maybe not. Let's avoid the term "span" in the filter name, since it naturally implies a span of time between a start and end time (rather than an HTML `<span>` element). `annotated_date` maybe?

Reply:
Makes sense. I appreciate that you care about such things!